src/Tooltip/TooltipTouch.tsx

```diff
@@ -80,40 +94,42 @@ export default class TooltipTouch extends React.PureComponent {
     });
   }
 
-  getLocals({ popover, ...props }) {
+  render() {
+    const { popover: _popover, children, className, id, style, type, size } = this.props as TooltipTouchDefaultedProps;
     const { isOpen } = this.state;
-
-    return {
-      ...props,
+    const popover = Object.assign({
+      ..._popover,
       isOpen,
-      touchListeners: {
-        onTouchStart: this.onTouchStart,
-        onTouchEnd: this.onTouchEnd,
-        onTouchMove: isOpen ? this.onTouchMove : undefined
-      },
-      popover: {
-        ...popover,
-        isOpen,
-        className: cx(popover.className, 'tooltip-touch')
-      },
-      spanStyle: isOpen ? {
-        display: 'hidden',
-        position: 'fixed',
-        top: -1000
-      } : undefined
+      className: cx(_popover.className, 'tooltip-touch')
+    });
+    const spanStyle: React.CSSProperties = {
```

Comment:
be aware: `spanStyle` was `undefined` if `!isOpen` before. Are you sure it's not introducing regressions?